package.json

```diff
@@ -28,30 +42,36 @@
   },
   "homepage": "https://github.com/milieuinfo/webcomponent-vl-ui-cookie-consent#readme",
   "dependencies": {
-    "vl-ui-util": "^2.0.5",
-    "vl-ui-core": "^2.0.0",
-    "vl-ui-button": "^1.3.1",
-    "vl-ui-checkbox": "^1.0.17",
-    "vl-ui-form-message": "^3.0.0",
+    "vl-ui-button": "^1.3.2",
+    "vl-ui-checkbox": "^1.0.18",
+    "vl-ui-core": "^2.2.7",
     "vl-ui-form-grid": "^1.0.12",
-    "vl-ui-modal": "^1.0.14"
+    "vl-ui-form-message": "^3.0.0",
+    "vl-ui-modal": "^1.0.15"
   },
   "peerDependencies": {
     "vl-ui-button": "^1.3.1",
-    "vl-ui-checkbox": "^1.0.17",
+    "vl-ui-checkbox": "github:milieuinfo/webcomponent-vl-ui-checkbox#UIG-271",
```

Comment:
2 keer checkbox als dependency?

Reply:
Eén keer als dependency en één keer als peer dependency. Indien er een dependency clash zou zijn, zou de vl-ui-checkbox in een node_modules folder binnen de vl-ui-cookie-consent geladen worden. Maar er kan maar één vl-ui-checkbox geserveerd worden (via absoluut pad of via node import). Om te voorkomen dat een andere versie gebruikt wordt dan verwacht, worden de dependencies ook toegevoegd aan de peer dependencies. De afnemer krijgt dan bij een npm install een waarschuwing indien er versie conflicten zouden zijn.

Ik heb de GitHub versie wel vervangen door de laatste release van vl-ui-checkbox.